docs/examples/repeated_regions.py

```diff
@@ -0,0 +1,19 @@
+from pypom import Page, Region
+from selenium.webdriver.common.by import By
+
+
+class Results(Page):
+    _result_locator = (By.CLASS_NAME, 'result')
+
+    @property
+    def results(self):
+        results_element = self.find_element(*self._result_locator)
```

Comment:
There's really no need for the `results_element` in this example. If we just want all results then our `_results_locator` should locate them all. With the existing HTML this could be:

```python
_result_locator = (By.CSS_SELECTOR, '.result.item')
```